### ISSUE TITLE
Problem with the response code

### ISSUE THREAD

**Issue by alokchaturvedi04** [shivprakash210/wp-service-payment-form-with-authorizenet#18]
you add response code "1 or 4 means the transaction was a success"
but in http://authorize.net/ only 1 response code for a successful transition.

what happens once we click the submit button when trying to process a payment from the form. It was working, but yesterday and today we tried putting in a credit card and once you hit pay now/submit, the form doesn’t notify you of the payment being processed. Now it stays on the same page and doesn’t go anywhere and no payment comes through.

**Comment by shivprakash210** [comment]
Yes, this issue is known issue and I am updating plugin. I hope I will completed it very soon.

**Comment by romondodavis** [comment]
I'm having this problem intermittently — the transaction is not processed and the user sees a blank page after submitting the form.
